docs/reference/connectors/gpubsub_producer.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_label: gpubsub_producer
```

Comment:
Would it make sense to add a note that gpubsub is google pub sub as we do for gcs?
```suggestion
sidebar_label: gpubsub_producer (Google Pub/Sub)
```